config/locales/en.yml

```diff
@@ -4,6 +4,7 @@ en:
   unauthorized: "HTTP Token: Access denied"
   bad_request: "Bad request"
   not_found: "Not found"
+  invalid_page_parameters: "The '#/page[page]' and '#/page[per_page]' parameter values must be a valid number (equal and more than 1)"
```

Comment:
"...must be a positive number" maybe?

"equal and more than 1" doesn't sound quite right.